src/multiselect/styles.scss

```diff
@@ -12,10 +12,7 @@
 
 .embedded {
   @include styles.styles-reset;
-
-  display: flex;
-  flex-direction: column;
-  inline-size: 100%;
+  display: contents;
```

Comment:
This is a small fix for embedded multiselect to be used in the property filter (already used in the test page).